### ISSUE TITLE
Move Prettier and ESLint config and scripts to /frontend folder

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#2726]
Instead of maintaining duplicates in every frontend subproject.

And switch json config to yml files to allow comments.

**Comment by tristan-orourke** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/GC-Digital-Talent-6092dc05157fa30010edaef9/issues/GCTC-NTGC/gc-digital-talent/2726?planning-poker) with ZenHub @petertgiles

**Comment by mnigh** [comment]
@esizer is this issue still relevant after #5065?

**Comment by esizer** [comment]
> @esizer is this issue still relevant after #5065?

I dont think so, we have the shared config files in the following locations now:

- [eslint-config-custom](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/eslint-config-custom)
- [jest-presets](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/jest-presets)
- [storybook-helpers](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/storybook-helpers)
- [tsconfig](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/tsconfig)
- [webpack-config](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/webpack-config)

**Comment by mnigh** [comment]
> > @esizer is this issue still relevant after #5065?
> 
> I dont think so, we have the shared config files in the following locations now:
> 
> * [eslint-config-custom](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/eslint-config-custom)
> * [jest-presets](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/jest-presets)
> * [storybook-helpers](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/storybook-helpers)
> * [tsconfig](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/tsconfig)
> * [webpack-config](https://github.com/GCTC-NTGC/gc-digital-talent/tree/main/packages/webpack-config)

@tristan-orourke any objection to closing this issue as stale?

**Comment by tristan-orourke** [comment]
Stale issue, the work is already done.
